content/docs/iac/concepts/resources/providers.md

```diff
@@ -50,7 +50,42 @@ For example, to generate a local SDK for the [`hashicorp/random` provider](https
 $ pulumi package add terraform-provider hashicorp/random
 ```
 
+{{% notes type="tip" %}}
 In order to make sure Pulumi users are aware of the Any Terraform Provider's capabilities, Pulumi has included select, popular providers that can be consumed in Pulumi via the Any Terraform Provider in the Pulumi Registry, such as [The Honeycomb provider](/registry/packages/honeycombio/).
+{{% /notes %}}
+
+The generated SDK will include a `.gitignore` so it can be safely committed to version control without including all of the SDK's dependencies. The SDK contains instructions to download the provider binary to a shared (and cached) location on your local system outside of the working directory - the binary is not committed to version control.
+
+#### Adding Provider Packages to the Project Configuration File
+
+{{% notes type="info" %}}
+Adding provider packages to your project configuration file requires Pulumi version 3.157.0 or later.
+{{% /notes %}}
+
+You can avoid the need to commit any generated SDK files to version control by adding your parameterized packages to your Pulumi project configuration file (`Pulumi.yaml`).
+
+You can add a list of your named provider packages under the [`packages`](/docs/iac/concepts/projects/project-file/#packages-options) key. For the example in the previous section, the syntax would be:
+
+```yaml
+packages:
+  random: # This is an arbitrary name
+    source: terraform-provider
+    parameters:
```

Comment:
Users can also specify a `version` of the provider and we should probable encourage doing this. Version is applicable both for `terraform-provider` and `hashicorp/random` (they are different versions).

Reply:
@mikhailshilkov Can you verify this is now correct?